apps/challenges/models.py

```diff
@@ -202,6 +203,8 @@ def create_eks_cluster_for_challenge(sender, instance, created, **kwargs):
         ):
             serialized_obj = serializers.serialize("json", [instance])
             aws.setup_eks_cluster.delay(serialized_obj)
+    if created:
+        construct_and_send_remote_evaluation_mail(instance)
```

Comment:
@burnerlee we should call this method from `create_challenge_using_zip` API in `views/challenges.py`. When a challenge is successfully uploaded using the API and challenge is a remote evaluation challenge then only we will send this email. You can check if a challenge is a remote_evaluation challenge using `remote_evaluation` field from challenge model